charts/datasaur/templates/config/intelligence-env-configMap.yaml

```diff
@@ -2,8 +2,21 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: intelligence-env
-data: 
+data:
   NLTK_PROVIDER_CORENLP_URL: http://{{ .Release.Name }}-core-nlp-service:9000
-  NLTK_PROVIDER_PORT: '9090'
+  NLTK_PROVIDER_PORT: "9090"
   SPARKNLP_JSL_HOST: http://{{ .Release.Name }}-jsl-server-service:5000
-  SPARKNLP_PORT: '9091'
\ No newline at end of file
+  SPARKNLP_PORT: "9091"
+
+  # Datasaur Dinamic environment variables
+  DATASAUR_ASSIST_VULCAN_BASIC_AUTH: {{ .Values.intelligence.datasaurDinamic.authKey }}
+  DATASAUR_ASSIST_VULCAN_HOST: http://{{ .Release.Name }}-backend-service:8080
+  # the credentials here are used to connect to object storage and assuming role for sagemaker access
+  # currently Datasaur Dinamic only support s3
+  DATASAUR_ASSIST_OBJECT_STORAGE_BUCKET: {{ .Values.intelligence.datasaurDinamic.objectStorageBucket }}
+  DATASAUR_ASSIST_OBJECT_STORAGE_BUCKET_PREFIX: {{ .Values.intelligence.datasaurDinamic.objectStorageBucketPrefix }}
+  DATASAUR_ASSIST_OBJECT_STORAGE_ENDPOINT:   {{ .Values.intelligence.datasaurDinamic.objectStorageEndpoint }}
+  DATASAUR_ASSIST_AWS_REGION:   {{ .Values.intelligence.datasaurDinamic.awsRegion }}
+  DATASAUR_ASSIST_AWS_ACCESS_KEY: {{ .Values.intelligence.datasaurDinamic.awsAccessKey }}
+  DATASAUR_ASSIST_AWS_SECRET_KEY:  {{ .Values.intelligence.datasaurDinamic.awsSecretKey }}
```

Comment:
please also update values.sample.yaml in helm scheme in our repository

Reply:
updated in values sample yaml